README.rst

```diff
@@ -22,13 +22,18 @@ OpenWISP 2 monitoring module (Work in progress).
 Available Features
 ------------------
 
-* Displays ``Uptime``, ``Memory Status``, ``Load Status``, ``Addresses``, ``Neighbours`` of the **Device**
-* Displays ``Monitoring Graphs`` namely ``Uptime``, ``Packet Loss``, ``Round Trip Time``, ``Traffic`` in the form of lively charts
-* ``Monitoring Graphs`` can be viewed at resolutions of 1 day, 3 days, a week, a month and a year
+* Collects and displays device status information like uptime, RAM memory status, CPU load averages,
+  Interface addresses, WiFi interface status and associated clients, Neighbors information, DHCP Leases, Disk/Flash status
+* Collection of monitoring information in a timeseries database (currently only influxdb is supported)
+* Monitoring charts for uptime, packet loss, round trip time (latency), associated wifi clients, interface traffic,
+  RAM memory usage, CPU load, flash/disk usage
```

Comment:
It might be better to simply use RAM usage/ RAM (memory) usage as RAM already has `memory` in the abbreviation so it sounds little strange, wdyt?

Reply:
done, thanks